components/atoms/a-images-grid.vue

```diff
@@ -52,10 +52,10 @@ export default {
   }
 }
 .a-images-grid {
-  max-width: 960px;
```

Comment:
This is not needed anymore, parent container should define proper width - not the atom itself. And this is already done right in our Capybara ;)

Reply:
thanks for change :)